target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/vr9_buffalo_wbmr-300hpd.dts

```diff
@@ -162,25 +162,25 @@
 		reg = <0>;
 		lantiq,switch;
 
-		ethernet@1 {
+		ethernet@4 {
```

Comment:
Same here; unless there is a reason for the current order I don't see, I'd order all the cases you touch here by reg number.

Reply:
Ah, I reviewed the patches one-by-one. I'm not really sure whether the third commit really needs to be separate, I'd probably have squashed it, but I don't have a strong preference for that, so do as you wish there.